source/gui/_addonStoreGui/viewModels/addonList.py

```diff
@@ -292,10 +301,14 @@ def _getSortFieldData(listItemVM: AddonListItemVM) -> "SupportsLessThan":
 		def _containsTerm(detailsVM: AddonListItemVM, term: str) -> bool:
 			term = term.casefold()
 			model = detailsVM.model
+			inPublisher = isinstance(model, _AddonStoreModel) and term in model.publisher.casefold()
+			inAuthor = isinstance(model, _InstalledAddonModel) and term in model.author.casefold()
 			return (

```

Comment:
Could we also search for IDs since it is now a displayed information (and was already an information sometimes seen by user e.g. in the uninstall dialog boxes)?

Since the ID is often similar to the add-on's name in English, it may be especially useful for non-English speakers used  though to know also English names of the add-ons.

Or maybe even better: for non-English users to be able to filter either on English add-on's name or non-English add-on's name. E.g. be able to filter on "Mise à Jour des Extensions" as well as on "Add-on Updater". If this is your choice, I will open a separate issue since it's beyond the scope of this PR.

Reply:
Searching via add-on ID has already been added just below in line 309